### DISCUSSION TITLE
Sync TickTick tasks to corresponding folders in Obsidian

### DISCUSSION THREAD

**Discussion by radhakrisri** [thesamim/TickTickSync#85] [Ideas]
My TickTick is organized as lists under different folders for e.g..

- Household
- Career
  - Certifications (list)
  - Trainings (list)
- Projects
  - Project 1 (list)
  - Project 2 (list)

etc.

I have a similar folder structure in Obsidian too. When I setup tickticksync on obsidian, I had to setup a separate folder where all the tasks in ticktick are synced to.

Would be great, if the lists are synced to corresponding folders in Obsidian.

**Comment by thesamim** [comment]
That's on my roadmap. There's a couple of architectural changes I have to make first. But we'll get there.
